src/nft/components/collection/Activity.tsx

```diff
@@ -138,9 +137,11 @@ export const Activity = ({ contractAddress, rarityVerified, collectionName, chai
                 (event, i) =>
                   event.eventType && (
                     <Box
-                      as="a"
+                      as={Link}
```

Comment:
Why is this changed to `Link` from `a`?

Reply:
Using `<Link />` lets us always refer to a path (`/`) and will use hashes or paths depending on deployment. Without it, and by relying on `<a />`, we would have to handle that case ourselves.

Reply:
In principle, we should always rely on `<link />` where possible.